packages/fw-avatar/test/fw-avatar-group.test.js

```diff
@@ -0,0 +1,247 @@
+  let element;
+  beforeEach(async () => {
+    element = await fixture(html` <fw-avatar-group
+      .items=${users}
+      nameAttribute=${nameAttribute}
+      secondaryAttribute=${secondaryAttribute}
+      .maxCount=${maxCount}
+      header=${header}
+      .showSearchBar=${showSearchBar}
+      .primaryAction=${primaryAction}
+      .secondaryAction=${secondaryAction}
+      emptyStateMessage=${emptyStateMessage}
+      .emptyStateAction=${emptyStateAction}
+      .absolute=${true}
+    ></fw-avatar-group>`);
+  });
+
+  it("FwAvatarGroup instance check", () => {
+    assert.instanceOf(
+      element,
+      FwAvatarGroup,
+      " The Element created from tag fw-avatar-group is not an instance of FWAvatarGroup class\n"
+    );
+  });
+
+  it("checks for items(with secondary attr) content", async () => {
+    const itemsListHtml = element.shadowRoot.querySelectorAll(
+      "mwc-list-item.items-list-item > span"
+    );
+
+    let tempArray = Array();
+    for (let i = 0; i < itemsListHtml.length; i = i + 2) {
+      tempArray.push({
+        name: String(itemsListHtml[i].textContent.trim()),
+        company: String(itemsListHtml[i + 1].textContent.trim()),
+      });
+    }
+    expect(element.items).to.equal(users);
+    expect(tempArray).to.be.eql(users);
+  });
+
+  it("checks for items(without secondary attr) content", async () => {
+    element.items = usersWithOutSecondaryAttribute;
+    element.removeAttribute("secondaryAttribute");
+    await element.elementUpdated;
+
+    const itemsListHtml = element.shadowRoot.querySelectorAll(
+      "mwc-list-item.items-list-item > span"
+    );
+
+    let tempArray = Array();
+    for (let i = 0; i < itemsListHtml.length; i = i + 1) {
+      tempArray.push({
+        name: String(itemsListHtml[i].textContent.trim()),
+      });
+    }
+    expect(element.items).to.equal(usersWithOutSecondaryAttribute);
+    expect(tempArray).to.be.eql(usersWithOutSecondaryAttribute);
+  });
+
+  it("checks Max Count property", async () => {
+    const visibleAvatars = element.shadowRoot
+      .querySelector("#group")
+      .querySelectorAll("fw-avatar[type='initials']");
+    expect(element.maxCount).to.equal(maxCount);
+    expect(visibleAvatars.length).to.equal(
+      Math.min(users.length, maxCount),
+      "The number of avatars rendered in the group do not match with MaxCount\n"
+    );
+  });
+
+  it("checks menu visiblility on tap/Click event", async () => {
+    expect(
+      element.shadowRoot.querySelector("#menu").getAttributeNames()
+    ).not.to.include("open", "Menu in open mode without Click event \n");
+
+    const avatarGroupButton = element.shadowRoot.querySelector(".group");
+
+    setTimeout(() => {
+      avatarGroupButton.click();
+    });
+    const {} = await oneEvent(avatarGroupButton, "click");
+
+    expect(
+      element.shadowRoot.querySelector("#menu").getAttributeNames()
+    ).to.include("open", "Click even didn't triggered the menu to  open \n");
```

Comment:
* `Click event didn't trigger the menu to open`

Reply:
done👍🏼